PurchasesTests/Mocks/MockHTTPClient.swift

```diff
@@ -0,0 +1,38 @@
+import Purchases
+
+class MockHTTPClient: RCHTTPClient {
```

Comment:
the mocks in these PRs are mostly autogenerated. It's nice because they also have very consistent formatting and naming, plus they're pretty complete.